<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guestBrigades</class>
 <widget class="QDialog" name="guestBrigades">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>538</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1236</width>
    <height>538</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1236</width>
    <height>538</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Наши бригады</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twOrg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teResult">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFind">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите название бригады...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFind">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twOrgFind">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>20</width>
     <height>541</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Бригады с которыми мы сотрудничаем:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Состав каждой бригады:</string>
   </property>
  </widget>
  <zorder>twOrg</zorder>
  <zorder>teResult</zorder>
  <zorder>btnBack</zorder>
  <zorder>leFind</zorder>
  <zorder>btnFind</zorder>
  <zorder>twOrgFind</zorder>
  <zorder>line</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
